### ISSUE TITLE
Setup common CI/CD environment (Azure DevOps)

### ISSUE THREAD

**Issue by devlace** [Azure-Samples/modern-data-warehouse-dataops#796]
We need a deployment pipeline that Continuously deploys our sample

Related: #797
## DoD:
- [ ] Setup an Azure DevOps project, same tenant as our Fabric environment
- [ ] Create a release pipeline that deploys our sample
- [ ] Access to Azure DevOps project managed via SG
- [ ] Document how gain access. Add this to our dev onboarding guide: aka.ms/dataops-repo-onboarding

**Comment by promisinganuj** [comment]
I don't think that it's worth having this at this point of time. Once we have application running with real-like data, we would incorporate this.
